### DISCUSSION TITLE
API Qfield Cloud response

### DISCUSSION THREAD

**Discussion by lecault** [opengisch/QField#3765] [Q&A]
Hello,

I want to connect with API to download data.

The file is : http://app.qfield.cloud/api/v1/files/41c33f14-7a15-455a-8867-46331f2c81be/arbreindividueltestqfield.gpkg/
I do a Authorization Basic / GET request but i have code 401.

Is there anything else to do ?

Thanks,
Loïc

**Comment by suricactus** [comment]
Hey @lecault , please consider using the official QFieldCloud SDK (https://github.com/opengisch/qfieldcloud-sdk-python) or find inspiration in it. Since QFieldCloud is very your project and still in the 0.x versioning, expect any other integration than the official one to suffer incompatible changes and do them on your own risk.

**Comment by lecault** [comment]
Thanks to your response.
I try his next year ;-).
